### ISSUE TITLE
Repository selection of new issues is very problematic

### ISSUE THREAD

**Issue by haikalpribadi** [codetree/feedback#280]
Hi,

We're using CodeTree to manage issues across multiple GitHub repositories - which I believe is what CodeTree is made for. :)

However, we find that the repository assignment of new issues is very problematic.

1. When we click on "new issue", the default repository is automatically selected, even though, we are on a page filter made for a different repository. I.e. when you're on a filter page for repo-B, you don't expect to have repo-A to be the default selected repo for the new issue.
2. When we close the "new issue" pop up, and re-open it again by clicking "new issue", the selected repository (along with other details) are from the previous popup that has been closed.

Number 1 and 2 makes it very error prone for us making new issues across many repositories. You might as well just leave repository selectin BLANK (and milestone and labels BLANK for that matter) every time you open the "new issue" popup.

This is a big problem because we don't have the ability to delete issues in GitHub, nor are we able to move them to a different repo.

Any thoughts on this? Thank you!

**Comment by rstocker99** [comment]
Agreed. This is super annoying. I've added your vote to the internal issue we have tracking this.
